plugins/mongodb-adapter/src/main/java/org/polypheny/db/adapter/mongodb/MongoPlugin.java

```diff
@@ -291,8 +291,6 @@ public void rollback( PolyXid xid ) {
 
         @Override
         public void shutdown() {
-            DockerContainer.getContainerByUUID( deploymentId ).ifPresent( DockerContainer::destroy );
```

Comment:
Why removing this?

Reply:
Because it is redundant and can cause `NullPointerException` when remote deployment is used.  The `Adapter`  takes care of this (see [here](https://github.com/polypheny/Polypheny-DB/blob/various-improvements/core/src/main/java/org/polypheny/db/adapter/Adapter.java#L151-L157)).